lib/express_validator.js

```diff
@@ -99,6 +99,19 @@ var expressValidator = function(options) {
     return this;
   };
 
+  ValidatorChain.prototype.withMessage = function(message) {
+    if (!!this.lastError) {
```

Comment:
No need for the double-negative boolean conversion since the `if` will already evaluate it as boolean.